draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2961,6 +2963,16 @@ This approach increases the Hamming distance between inputs to different
 invocations of H, which reduces the likelihood that nonidealities in H
 affect the distribution of the b\_i values.
 
+## Target Security Levels {#security-considerations-targets}
+
```

Comment:
A small clash of notation, that can be easily solved, appears on Section 6.8.1 Rational Maps
```
Analogous ambiguities in other standardized rational maps MUST be resolved in the same way: for any constant k whose sign is ambiguous, k MUST be chosen such that sgn0(k) == 1.
```

Reply:
Also in Section 8.8 Defining a new hash curve. 
says: 
`k` is the security level of the elliptic curve.
should say:
`k` is the targeted security level of the suite.

Reply:
but I'd suggest "target" rather than "targeted"

For section 6.8.1, maybe use `const`?

Reply:
Great catch!

Reply:
> For section 6.8.1, maybe use const?

I was just going to use `c`, since that's what we use for constants.